e2e/e2e_tests.py

```diff
@@ -39,105 +46,119 @@ def _stop_containers():
     docker_compose.clean()
 
 
-class TestE2E(unittest.TestCase):
-    COOKIES = dict()
-    CSRF_TOKEN = ''
-
-    def _login(self, next=None):
-        login_url = self._make_link('/login/')
-        client = requests.session()
-        client.get(login_url)
+class SeleniumTests(unittest.TestCase):
+    # wait maximal 5 minutes
+    wait_time_seconds = 5*60
+    excerpt_data = {
+        'send_keys': {
+            'new_excerpt_name': 'HSR',
+            'new_excerpt_bounding_box_north': '47.22407852727801',
+            'new_excerpt_bounding_box_west':  '8.815616369247437',
+            'new_excerpt_bounding_box_east':  '8.819221258163452',
+            'new_excerpt_bounding_box_south': '47.222388077452706',
+        },
+        'click': [
+            'export_options.excerptconverter.gisexcerptconverter.gis_excerpt_converter.formats_0',
+            'export_options.excerptconverter.gisexcerptconverter.gis_excerpt_converter.formats_1',
+            'export_options.excerptconverter.gisexcerptconverter.gis_excerpt_converter.formats_2',
+            'export_options.excerptconverter.gisexcerptconverter.gis_excerpt_converter.formats_3',
+        ],
+    }
+    driver = webdriver.Firefox
+
+    def setUp(self):
+        self.browser = self.driver()
+
+    def tearDown(self):
+        self.browser.close()
+
+    def test_login_logout(self):
+        self._login()
+        self._logout()
 
-        csrf_token = client.cookies['csrftoken']
-        login_data = {
-            'password': ADMIN_PASSWORD_FOR_TESTS,
-            'username': ADMIN_USER_FOR_TESTS,
-            'csrfmiddlewaretoken': csrf_token,
-            'next': next,
-        }
-        r = client.post(login_url, data=login_data)
-        return {'request': r, 'client': client}
+    def test_create_new_excerpt_succeeds(self):
+        self._login()
+        self._create_new_order()
+        self._go_to_order()
+        links = self._wait_for_order_to_complete_and_fetch_download_links()
+        self.assertEqual(len(links), 8)
+        download_links = self._get_download_links_from_links(links)
```

Comment:
Method naming here makes not as much sense as it could. As written now it seems like
- You fetch download links and put them into variable `links`
- (you assert that there are 8 of them)
- you get those links from `links` which are download links (which should be all of them, as 'download links' is what you've put in there above) and put them into variable `download_links`.